### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by KristianKjerstad**
- round bridge values to one decimal
- added total score to radar chart
- change default pill volume to 1
- sum density for concentration blends

### CODE REVIEW

File: api/calculators/bridge.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def calculate_blend_cumulative(product_list: List[Product]):
             for i, _ in enumerate(SIZE_STEPS):
                 cumulative_curve[i] += product.share * product.cumulative[i]
 
-    return cumulative_curve
+    return [round(num, 1) for num in cumulative_curve]
```
Comment (soofstad) [inline api/calculators/bridge.py:42]: How did this effect performance?
This is run in the fitness function (eg. many times)
Better to do this in the final response step

File: web/src/Components/Optimization/OptimizationRunner.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ const OptimizationRunner = ({ mode, value, handleUpdate, allProducts }: Optimiza
   const [iterations, setIterations] = useState<number>(2000)
   const [maxProducts, setMaxProducts] = useState<number>(5)
   const [pill, setPill] = useState<Pill>({
-    volume: 10,
+    volume: 1,
     density: 350,
     mass: 3500,
```
Comment (soofstad) [inline web/src/Components/Optimization/OptimizationRunner.tsx:56]: This should be 350 now

**Review by soofstad — CHANGES_REQUESTED** [review]
A few minor.
After that good to merge :+1:

**Comment by KristianKjerstad** [comment]
closes #101 #103  #104 #105

**Pull request merged**